src/admin/dumb_components/letters/letter_recipients.js

```diff
@@ -1,11 +1,15 @@
 import React from 'react'
 
-export default ({ letters }) =>
+export default ({ letters, show_letter, clickable }) =>
   <div className='letter-recipients'>
     <h2>Letter Recipients</h2>
     <ul>
       {letters.map((letter, i) =>
-        <li key={letter.first_name + i}>
+          <li
+            onClick={clickable ? () => show_letter(i) : null}
+            className={clickable ? 'clickable-letter-recipient' : ''}
+            key={letter.first_name + i}
```

Comment:
Do we have access to the member id? That would be a preferable key. It doesn't matter too much as we won't be updating this list too rapidly